### PR TITLE
Dynamic Icons For OEM Calendar Apps

### PR DESCRIPTION
**Pull request by naveenccmsd**
## Description
Dynamic Icons For OEM Calendar Apps

Fixes #(issue) 

## Type of change

:x: General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: lawnchair/res/values/config.xml [hunk 1]
```diff
@@ -18,6 +18,18 @@
...
+        <item>com.xiaomi.calendar</item>
+        <item>com.samsung.android.calendar</item>
+        <item>com.blackberry.calendar</item>
+        <item>com.huawei.calendar</item>
+        <item>com.simplemobiletools.calendar.pro</item>
+        <item>org.lineageos.etar</item>
+        <item>ws.xsoh.etar</item>
+        <item>org.withouthat.acalendarplus</item>
+        <item>org.withouthat.acalendar</item>
+    </string-array>
```
Comment (Goooler) [inline lawnchair/res/values/config.xml]: Need to resort? And you can also add `com.oneplus.calendar` to handle the Oneplus calendar.

**Comment by Goooler** [comment]
Submodule changes also need to be sent a PR to https://github.com/LawnchairLauncher/platform_frameworks_libs_systemui

**Comment by naveenccmsd** [comment]
Created PR for submodule changes https://github.com/LawnchairLauncher/platform_frameworks_libs_systemui/pull/10

**Comment by daywalk3r666** [comment]
Maybe the same thing can do with clock icon?

**Comment by naveenccmsd** [comment]
Co-authored-by: @daywalk3r666

**Comment by naveenccmsd** [comment]
> Maybe the same thing can do with clock icon?

I tried bro , it was taking some time. I will open another PR when it ready. 
You have have draft changes on the same https://github.com/naveenccmsd/lawnchair/tree/comp.

**Comment by yasanglass** [comment]
I do not have access to all the calendar apps, are they all tested and properly working?

**Comment by daywalk3r666** [comment]
For me i can confirm acalendar and aosp is working.

**Comment by daywalk3r666** [comment]
Screenshot_20221101-092852_Lawnchair

**Comment by naveenccmsd** [comment]
I have tested on Mi Calendar and Samsung calendar . It is working.

**Comment by naveenccmsd** [comment]
But there shouldn't be any static calendar added in lawnicons, For now samsuncg calendar present in lawnincons.

**Comment by naveenccmsd** [comment]
Screenshot_20221101-141716_Lawnchair

**Comment by yasanglass** [comment]
Okay great, it worked fine on my tests as well.

**Review by yasanglass — APPROVED** [review]

**Comment by yasanglass** [comment]
@naveenccmsd thank you for contributing to Lawnchair!

**Pull request merged**